<?xml version="1.0" ?>
<launch>

	<remap from="/study/depth/points"
		to="/camera/depth_registered/points"/>
	<remap from="/study/rgb/image"
		to="/camera/rgb/image_rect_color"/>


  	<node name="rviz" pkg="rviz" type="rviz" args="-d /home/taylor/.rviz/lifting_eye.rviz"/>
  	<node name="picked_segmentation" pkg="pick" type="picked_segmentation" output="screen"/>
  	<node name="picked_icp" pkg="pick" type="picked_icp" output="screen"/>
  	<!--node pkg="dynamic_reconfigure" type="dynparam" name="dynparam" args="set /camera/driver depth_registration true" /-->
</launch>

